clients/admin-ui/src/features/dataset/EditCollectionDrawer.tsx

```diff
@@ -28,6 +28,15 @@ const EditCollectionDrawer = ({ collection, isOpen, onClose }: Props) => {
   const collectionIndex = useSelector(selectActiveCollectionIndex);
   const [updateDataset] = useUpdateDatasetMutation();
   const toast = useToast();
+  const deleteMessage = (
+    <Text>
+      You are about to permanently delete the collection named{" "}
+      <Text color="complimentary.500" as="span" fontWeight="bold">
```

Comment:
More of a design question, but the blue kinda makes this look like a clickable link. 🤔

Reply:
Yeah, I agree... I'll leave a comment in the figma!